build/parse.js

```diff
@@ -77,8 +75,8 @@ exports.split = function(string) {
   regex += '\\S+';
   result = string.match(new RegExp(regex, 'g')) || [];
   return _.map(result, function(word) {
-    word = _.str.unquote(word, '\'');
-    word = _.str.unquote(word, '"');
+    word = word.replace(/^'(.*)'$/, '$1');
+    word = word.replace(/^"(.*)"$/, '$1');
```

Comment:
Would `word = word.replace(/^('|")(.*)\1$/, '$2');` be better here?

Reply:
It might be but the behaviour would be different to currently and maybe break some edge case, which I wanted to avoid. That said I do think only removing the first set of quotes would be the right choice if not for compatibility concerns